pkg/controller/replicaset/replica_set_test.go

```diff
@@ -443,9 +445,9 @@ func TestSyncReplicaSetDormancy(t *testing.T) {
 	manager.syncReplicaSet(getKey(rsSpec, t))
 	validateSyncReplicaSet(t, &fakePodControl, 1, 0, 0)
 
-	// 1 PUT for the ReplicaSet status during dormancy window.
+	// 2 PUT for the ReplicaSet status during dormancy window.
 	// Note that the pod creates go through pod control so they're not recorded.
-	fakeHandler.ValidateRequestCount(t, 1)
+	fakeHandler.ValidateRequestCount(t, 2)
```

Comment:
There is an extra update in the RS with a ReplicaFailure condition because of the fake error that is set in the pod client.